src/main/java/com/minecolonies/blocks/BlockHutFarmer.java

```diff
@@ -1,4 +1,16 @@
 package com.minecolonies.blocks;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.tileentities.TileEntityColonyBuilding;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.tileentities.TileEntityColonyBuilding'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)